src/rosconsole/impl/rosconsole_log4cxx.cpp

```diff
@@ -354,6 +354,12 @@ void register_appender(LogAppender* appender)
   logger->addAppender(g_log4cxx_appender);
 }
 
+void deregister_appender(){
+	const log4cxx::LoggerPtr& logger = log4cxx::Logger::getLogger(ROSCONSOLE_ROOT_LOGGER_NAME);
+ 	logger->removeAppender(g_log4cxx_appender);
+	delete g_log4cxx_appender;
+	g_log4cxx_appender = 0;
```

Comment:
Please follow the same style of the file: 2 space indentation (no tabs).

Same below.